src/index.html

```diff
@@ -9,10 +10,57 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
```

Comment:
Ensure that the `style.css` file is separated into individual files for each BEM block as per the task requirements. This will help maintain modularity and organization in your styles.